/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.6.171:35260'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.6.171:39264'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.6.171:37535'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.6.171:37530'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.6.171:46784'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.6.171:42578'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.6.171:32956'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.6.171:40280'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.6.171:33725'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.6.171:40833'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.6.171:45236'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.6.171:46072'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.6.171:33642'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.6.171:34136'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.6.171:40045'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.6.171:34674'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.6.171:33787'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.6.171:41416'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.6.171:46871'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.6.171:36582'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.6.171:34892'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.6.171:40702'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.6.171:43270'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.6.171:39354'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.6.171:39625'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.6.171:42710'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.6.171:45524'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.6.171:40369'
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.6.171:39665
distributed.worker - INFO -          Listening to:   tcp://172.30.6.171:39665
distributed.worker - INFO -              nanny at:         172.30.6.171:39264
distributed.worker - INFO -              bokeh at:         172.30.6.171:40254
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-mhqj560x
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.6.171:32804
distributed.worker - INFO -          Listening to:   tcp://172.30.6.171:32804
distributed.worker - INFO -              nanny at:         172.30.6.171:37535
distributed.worker - INFO -              bokeh at:         172.30.6.171:36352
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-n0239sh3
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.6.171:42075
distributed.worker - INFO -          Listening to:   tcp://172.30.6.171:42075
distributed.worker - INFO -              nanny at:         172.30.6.171:34892
distributed.worker - INFO -              bokeh at:         172.30.6.171:38814
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-av_zygmo
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.6.171:41119
distributed.worker - INFO -          Listening to:   tcp://172.30.6.171:41119
distributed.worker - INFO -              nanny at:         172.30.6.171:37530
distributed.worker - INFO -              bokeh at:         172.30.6.171:32909
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-v4_54my_
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.6.171:42849
distributed.worker - INFO -          Listening to:   tcp://172.30.6.171:42849
distributed.worker - INFO -              nanny at:         172.30.6.171:40280
distributed.worker - INFO -              bokeh at:         172.30.6.171:39335
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-e77p3663
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.6.171:43860
distributed.worker - INFO -          Listening to:   tcp://172.30.6.171:43860
distributed.worker - INFO -              nanny at:         172.30.6.171:40702
distributed.worker - INFO -              bokeh at:         172.30.6.171:37208
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-xgvfh002
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.6.171:42044
distributed.worker - INFO -          Listening to:   tcp://172.30.6.171:42044
distributed.worker - INFO -              nanny at:         172.30.6.171:34674
distributed.worker - INFO -              bokeh at:         172.30.6.171:40883
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-lmev7zc0
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.6.171:38603
distributed.worker - INFO -          Listening to:   tcp://172.30.6.171:38603
distributed.worker - INFO -              nanny at:         172.30.6.171:39625
distributed.worker - INFO -              bokeh at:         172.30.6.171:40293
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-myjultg3
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.6.171:45279
distributed.worker - INFO -          Listening to:   tcp://172.30.6.171:45279
distributed.worker - INFO -              nanny at:         172.30.6.171:46784
distributed.worker - INFO -              bokeh at:         172.30.6.171:38374
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-0iyg6rqx
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.6.171:35752
distributed.worker - INFO -          Listening to:   tcp://172.30.6.171:35752
distributed.worker - INFO -              nanny at:         172.30.6.171:32956
distributed.worker - INFO -              bokeh at:         172.30.6.171:36869
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-trs0i3kl
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.6.171:39701
distributed.worker - INFO -          Listening to:   tcp://172.30.6.171:39701
distributed.worker - INFO -              nanny at:         172.30.6.171:35260
distributed.worker - INFO -              bokeh at:         172.30.6.171:46778
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-gjj8oxe9
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.6.171:42970
distributed.worker - INFO -          Listening to:   tcp://172.30.6.171:42970
distributed.worker - INFO -              nanny at:         172.30.6.171:33787
distributed.worker - INFO -              bokeh at:         172.30.6.171:35215
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-0czj_9b8
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.6.171:44528
distributed.worker - INFO -          Listening to:   tcp://172.30.6.171:44528
distributed.worker - INFO -              nanny at:         172.30.6.171:40045
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.6.171:45247
distributed.worker - INFO -          Listening to:   tcp://172.30.6.171:45247
distributed.worker - INFO -              nanny at:         172.30.6.171:33642
distributed.worker - INFO -              bokeh at:         172.30.6.171:42669
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40264
distributed.worker - INFO -              bokeh at:         172.30.6.171:40941
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-0xvh5h2n
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-_vi694by
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.6.171:40105
distributed.worker - INFO -          Listening to:   tcp://172.30.6.171:40105
distributed.worker - INFO -              nanny at:         172.30.6.171:46871
distributed.worker - INFO -              bokeh at:         172.30.6.171:39612
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-1lqzynkz
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.6.171:41033
distributed.worker - INFO -          Listening to:   tcp://172.30.6.171:41033
distributed.worker - INFO -              nanny at:         172.30.6.171:42710
distributed.worker - INFO -              bokeh at:         172.30.6.171:37480
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-09ojs1t5
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.6.171:34771
distributed.worker - INFO -          Listening to:   tcp://172.30.6.171:34771
distributed.worker - INFO -              nanny at:         172.30.6.171:42578
distributed.worker - INFO -              bokeh at:         172.30.6.171:36385
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-u8zsaexw
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.6.171:45224
distributed.worker - INFO -          Listening to:   tcp://172.30.6.171:45224
distributed.worker - INFO -              nanny at:         172.30.6.171:39354
distributed.worker - INFO -              bokeh at:         172.30.6.171:36472
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-va2kknfb
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.6.171:42560
distributed.worker - INFO -          Listening to:   tcp://172.30.6.171:42560
distributed.worker - INFO -              nanny at:         172.30.6.171:46072
distributed.worker - INFO -              bokeh at:         172.30.6.171:41104
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-x_l39h5u
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.6.171:36432
distributed.worker - INFO -          Listening to:   tcp://172.30.6.171:36432
distributed.worker - INFO -              nanny at:         172.30.6.171:40833
distributed.worker - INFO -              bokeh at:         172.30.6.171:46446
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-we3a0x98
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.6.171:34879
distributed.worker - INFO -          Listening to:   tcp://172.30.6.171:34879
distributed.worker - INFO -              nanny at:         172.30.6.171:41416
distributed.worker - INFO -              bokeh at:         172.30.6.171:34701
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-pvw5rjn9
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.6.171:44490
distributed.worker - INFO -          Listening to:   tcp://172.30.6.171:44490
distributed.worker - INFO -              nanny at:         172.30.6.171:34136
distributed.worker - INFO -              bokeh at:         172.30.6.171:43313
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-wxo8fpbi
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.6.171:45735
distributed.worker - INFO -          Listening to:   tcp://172.30.6.171:45735
distributed.worker - INFO -              nanny at:         172.30.6.171:33725
distributed.worker - INFO -              bokeh at:         172.30.6.171:34432
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-df58cam4
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.6.171:44631
distributed.worker - INFO -          Listening to:   tcp://172.30.6.171:44631
distributed.worker - INFO -              nanny at:         172.30.6.171:36582
distributed.worker - INFO -              bokeh at:         172.30.6.171:41531
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-k0_bck70
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.6.171:43698
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.6.171:35122
distributed.worker - INFO -          Listening to:   tcp://172.30.6.171:43698
distributed.worker - INFO -          Listening to:   tcp://172.30.6.171:35122
distributed.worker - INFO -              nanny at:         172.30.6.171:45524
distributed.worker - INFO -              nanny at:         172.30.6.171:45236
distributed.worker - INFO -              bokeh at:         172.30.6.171:40646
distributed.worker - INFO -              bokeh at:         172.30.6.171:33995
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40264
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-pd12y1ei
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-k5w4ksfr
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.6.171:46695
distributed.worker - INFO -          Listening to:   tcp://172.30.6.171:46695
distributed.worker - INFO -              nanny at:         172.30.6.171:40369
distributed.worker - INFO -              bokeh at:         172.30.6.171:38069
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-1ny3bopk
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.6.171:41980
distributed.worker - INFO -          Listening to:   tcp://172.30.6.171:41980
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -              nanny at:         172.30.6.171:43270
distributed.worker - INFO -              bokeh at:         172.30.6.171:33362
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-di57yuso
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40264
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40264
distributed.core - INFO - Starting established connection
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40264
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40264
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40264
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40264
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40264
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40264
distributed.core - INFO - Starting established connection
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40264
distributed.core - INFO - Starting established connection
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40264
distributed.core - INFO - Starting established connection
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40264
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40264
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.core - INFO - Event loop was unresponsive in Worker for 3.06s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
distributed.core - INFO - Event loop was unresponsive in Worker for 163.15s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
distributed.core - INFO - Event loop was unresponsive in Worker for 163.34s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
distributed.core - INFO - Event loop was unresponsive in Worker for 163.15s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
distributed.core - INFO - Event loop was unresponsive in Worker for 162.87s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
distributed.core - INFO - Event loop was unresponsive in Worker for 162.88s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
distributed.worker - WARNING -  Compute Failed
Function:  execute_task
args:      ((subgraph_callable, (subgraph_callable, (<built-in function getitem>, (<function concatenate3 at 0x2ba396ccb048>, (<function concrete at 0x2ba313031a60>, [[[[array([[[[-0.05633001, -0.04627053, -0.03675834, ..., -0.07910673,
          -0.08060128, -0.07793941],
         [-0.06023889, -0.05275789, -0.04482593, ..., -0.07865693,
          -0.08148792, -0.07981391],
         [-0.0653514 , -0.06053206, -0.05303513, ..., -0.07900272,
          -0.08282983, -0.08228026],
         ...,
         [ 0.19121675,  0.18054226,  0.16526857, ..., -0.00396166,
          -0.00246525, -0.0051074 ],
         [ 0.19234695,  0.18679164,  0.17444369, ..., -0.00369334,
          -0.00139459, -0.00257758],
         [ 0.18898112,  0.18888775,  0.18070503, ..., -0.00376221,
          -0.00203443, -0.00138055]]]], dtype=float32), array([[[[-7.06561357e-02, -5.90874813e-02, -4.35932763e-02, ...,
           3.41969132e-02,  3.66391912e-02,  3.76573280e-02],
         [-7.33290166e-02, -6.23798594e-02, -4.67946045e
kwargs:    {}
Exception: TimeoutError('Timeout')

distributed.worker - WARNING -  Compute Failed
Function:  execute_task
args:      ((subgraph_callable, (subgraph_callable, (<built-in function getitem>, (<function concatenate3 at 0x2ba39694ed90>, (<function concrete at 0x2ba313031a60>, [[[[array([[[[-0.09778437, -0.10104617, -0.10387431, ..., -0.03873788,
          -0.03894789, -0.03902616],
         [-0.0958569 , -0.09884146, -0.10122781, ..., -0.03477461,
          -0.03490776, -0.03502205],
         [-0.09399295, -0.09659976, -0.09850761, ..., -0.0309579 ,
          -0.03097315, -0.03112036],
         ...,
         [-0.05077316, -0.04708818, -0.04364229, ..., -0.00530612,
           0.00559645,  0.01544577],
         [-0.05081489, -0.04689088, -0.04321611, ..., -0.00420839,
           0.0063837 ,  0.01593461],
         [-0.05104708, -0.04689229, -0.04300638, ..., -0.00291083,
           0.00736512,  0.01656176]]]], dtype=float32), array([[[[-0.03835916, -0.03698394, -0.0348911 , ...,  0.01601669,
           0.02591126,  0.03619171],
         [-0.03437079, -0.03298411, -0.03100534, ...,  0.01223762,
           0.
kwargs:    {}
Exception: TimeoutError('Timeout')

distributed.worker - WARNING -  Compute Failed
Function:  execute_task
args:      ((subgraph_callable, (subgraph_callable, (<built-in function getitem>, (<function concatenate3 at 0x2ba3969530d0>, (<function concrete at 0x2ba313031a60>, [[[[array([[[[-0.08122163, -0.07551929, -0.06949722, ...,  0.00878408,
           0.00647219,  0.00419235],
         [-0.09670075, -0.09164686, -0.08626577, ..., -0.01572095,
          -0.01791669, -0.02005351],
         [-0.1125203 , -0.10824908, -0.10370497, ..., -0.04098854,
          -0.04286529, -0.04468159],
         ...,
         [-0.11537819, -0.1162874 , -0.11683398, ..., -0.09551453,
          -0.09127347, -0.08667992],
         [-0.10091163, -0.10171871, -0.10225534, ..., -0.09012354,
          -0.0860019 , -0.081229  ],
         [-0.08711505, -0.08796614, -0.08857648, ..., -0.08525148,
          -0.08057419, -0.07502511]]]], dtype=float32), array([[[[ 0.00195024, -0.00027142, -0.00248972, ..., -0.11996652,
          -0.12656476, -0.12527366],
         [-0.02214737, -0.02422438, -0.02631254, ..., -0.11225247,
          -0.
kwargs:    {}
Exception: TimeoutError('Timeout')

distributed.worker - WARNING -  Compute Failed
Function:  execute_task
args:      ((subgraph_callable, (subgraph_callable, (<built-in function getitem>, (<function concatenate3 at 0x2ba396c54840>, (<function concrete at 0x2ba313031a60>, [[[[array([[[[ 0.00478595,  0.00343198,  0.00190351, ..., -0.16308573,
          -0.1567578 , -0.14988922],
         [ 0.0034637 ,  0.00190792,  0.00021828, ..., -0.15684341,
          -0.14961237, -0.14270417],
         [ 0.00199744,  0.00027697, -0.00147692, ..., -0.14943716,
          -0.14189537, -0.13546066],
         ...,
         [-0.05982142, -0.05957769, -0.05927072, ..., -0.02475682,
          -0.02132856, -0.01802154],
         [-0.06128727, -0.06089865, -0.06038262, ..., -0.02057642,
          -0.01721477, -0.01395519],
         [-0.06294983, -0.06252806, -0.06190529, ..., -0.01619107,
          -0.01283728, -0.00946599]]]], dtype=float32), array([[[[-0.14352468, -0.13789016, -0.1329291 , ..., -0.21435437,
          -0.22619571, -0.23526138],
         [-0.13682097, -0.13172245, -0.12709971, ..., -0.21673518,
          -0.
kwargs:    {}
Exception: TimeoutError('Timeout')

distributed.worker - WARNING -  Compute Failed
Function:  execute_task
args:      ((subgraph_callable, (subgraph_callable, (<built-in function getitem>, (<function concatenate3 at 0x2ba396d721e0>, (<function concrete at 0x2ba313031a60>, [[[[array([[[[-6.77627549e-02, -6.82346970e-02, -6.86013997e-02, ...,
          -4.21870425e-02, -4.13996726e-02, -4.09304686e-02],
         [-6.81631342e-02, -6.86177164e-02, -6.89795837e-02, ...,
          -4.27947864e-02, -4.21626791e-02, -4.17943373e-02],
         [-6.86601624e-02, -6.90870211e-02, -6.94401264e-02, ...,
          -4.39272188e-02, -4.33391295e-02, -4.29716483e-02],
         ...,
         [-3.67559046e-01, -3.68772715e-01, -3.69276762e-01, ...,
          -4.86535765e-02, -4.52107079e-02, -4.10607383e-02],
         [-1.54937031e-23,  5.88848575e-24,  7.25834074e-24, ...,
           1.01410475e-23, -8.74838619e-24,  5.39598845e-23],
         [ 0.00000000e+00,  0.00000000e+00,  0.00000000e+00, ...,
           0.00000000e+00,  0.00000000e+00,  0.00000000e+00]]]],
      dtype=float32), array([[[[-4.0790021e-02, -4.09694
kwargs:    {}
Exception: TimeoutError('Timeout')

distributed.worker - WARNING -  Compute Failed
Function:  execute_task
args:      ((subgraph_callable, (subgraph_callable, (<built-in function getitem>, (<function concatenate3 at 0x2ba396ca2268>, (<function concrete at 0x2ba313031a60>, [[[[array([[[[-6.1241742e-02, -6.1393689e-02, -6.1495423e-02, ...,
          -4.1487530e-02, -4.2320132e-02, -4.3282911e-02],
         [-6.1660532e-02, -6.1820962e-02, -6.1898213e-02, ...,
          -4.2308725e-02, -4.3031834e-02, -4.3874457e-02],
         [-6.2180933e-02, -6.2358949e-02, -6.2422894e-02, ...,
          -4.3461226e-02, -4.4068672e-02, -4.4784453e-02],
         ...,
         [-3.6300990e-01, -3.5509673e-01, -3.4732562e-01, ...,
          -3.1528454e-02, -3.0516820e-02, -2.8504912e-02],
         [ 7.5336406e-24, -2.6638498e-23, -1.6221598e-23, ...,
          -2.3098638e-24, -4.8688862e-24,  1.9621397e-22],
         [ 0.0000000e+00,  0.0000000e+00,  0.0000000e+00, ...,
           0.0000000e+00,  0.0000000e+00,  0.0000000e+00]]]],
      dtype=float32), array([[[[-4.4362973e-02, -4.5548201e-02, -4.6824623e-02, ...,
       
kwargs:    {}
Exception: TimeoutError('Timeout')

distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.6.171:57248 remote=tcp://172.30.100.1:40264>
distributed.worker - WARNING -  Compute Failed
Function:  execute_task
args:      ((subgraph_callable, (subgraph_callable, (<built-in function getitem>, (<function concatenate3 at 0x2ba39683f048>, (<function concrete at 0x2ba313031a60>, [[[[array([[[[-4.03847545e-03, -2.71373335e-03, -1.74572296e-03, ...,
          -1.28892632e-02, -1.43685993e-02, -1.58174299e-02],
         [-4.21781885e-03, -2.14364775e-03, -3.76523589e-04, ...,
          -1.41188558e-02, -1.58548709e-02, -1.75805558e-02],
         [-5.76300407e-03, -3.41025088e-03, -1.20075117e-03, ...,
          -1.58143360e-02, -1.79250855e-02, -1.99975055e-02],
         ...,
         [ 1.29675508e-01,  1.12231307e-01,  9.93227661e-02, ...,
           5.98166995e-02,  6.10046200e-02,  6.32421896e-02],
         [-1.82759587e-24, -5.79729738e-24, -1.25775486e-23, ...,
          -1.00761030e-23, -6.96607014e-24, -3.01871270e-22],
         [ 0.00000000e+00,  0.00000000e+00,  0.00000000e+00, ...,
           0.00000000e+00,  0.00000000e+00,  0.00000000e+00]]]],
      dtype=float32), array([[[[-1.71803776e-02, -1.8322
kwargs:    {}
Exception: TimeoutError('Timeout')

distributed.worker - WARNING -  Compute Failed
Function:  execute_task
args:      ((subgraph_callable, (subgraph_callable, (<built-in function getitem>, (<function concatenate3 at 0x2ba396cb4400>, (<function concrete at 0x2ba313031a60>, [[[[array([[[[-0.06112307, -0.05976964, -0.05831944, ...,  0.00967739,
           0.01086715,  0.01168865],
         [-0.06085287, -0.05948647, -0.05788392, ...,  0.00895614,
           0.01017858,  0.01113555],
         [-0.06002823, -0.0588276 , -0.05723482, ...,  0.00787073,
           0.00917794,  0.01053167],
         ...,
         [-0.03341044, -0.03308656, -0.03245353, ...,  0.00830203,
           0.00602105,  0.00459078],
         [-0.06508561, -0.06349844, -0.0618505 , ..., -0.00728449,
          -0.00707538, -0.00688494],
         [ 0.        ,  0.        ,  0.        , ...,  0.        ,
           0.        ,  0.        ]]]], dtype=float32), array([[[[ 0.01213887,  0.01247862,  0.01305132, ...,  0.        ,
           0.        ,  0.        ],
         [ 0.0120257 ,  0.01330784,  0.01550549, ...,  0.        ,
           0.
kwargs:    {}
Exception: TimeoutError('Timeout')

distributed.worker - WARNING -  Compute Failed
Function:  execute_task
args:      ((subgraph_callable, (subgraph_callable, (<built-in function getitem>, (<function concatenate3 at 0x2ba396c9bf28>, (<function concrete at 0x2ba313031a60>, [[[[array([[[[ 0.03802716,  0.03446672,  0.02418655, ..., -0.01112718,
          -0.00456693, -0.00029151],
         [ 0.03213435,  0.0361785 ,  0.03179524, ..., -0.00949084,
          -0.00347742,  0.0002925 ],
         [ 0.01949868,  0.03161058,  0.03374326, ..., -0.00762578,
          -0.00212131,  0.00105121],
         ...,
         [ 0.01362617,  0.0136509 ,  0.01431327, ..., -0.0448512 ,
          -0.04094678, -0.03203331],
         [ 0.01810975,  0.0176089 ,  0.01762199, ..., -0.04381057,
          -0.0446079 , -0.04077713],
         [ 0.02178484,  0.02161337,  0.02127909, ..., -0.03888889,
          -0.04174921, -0.04193827]]]], dtype=float32), array([[[[ 0.00172359,  0.00133342, -0.00134618, ...,  0.        ,
           0.        ,  0.        ],
         [ 0.00190212,  0.00135553, -0.0011942 , ...,  0.        ,
           0.
kwargs:    {}
Exception: TimeoutError('Timeout')

distributed.worker - WARNING -  Compute Failed
Function:  execute_task
args:      ((subgraph_callable, (subgraph_callable, (<built-in function getitem>, (<function concatenate3 at 0x2ba396c7be18>, (<function concrete at 0x2ba313031a60>, [[[[array([[[[1., 1., 1., ..., 1., 1., 1.],
         [1., 1., 1., ..., 1., 1., 1.],
         [1., 1., 1., ..., 1., 1., 1.],
         ...,
         [1., 1., 1., ..., 1., 1., 1.],
         [1., 1., 1., ..., 1., 1., 1.],
         [1., 1., 1., ..., 1., 1., 1.]]]]), array([[[[1., 1., 1., ..., 1., 1., 1.],
         [1., 1., 1., ..., 1., 1., 1.],
         [1., 1., 1., ..., 1., 1., 1.],
         ...,
         [1., 1., 1., ..., 1., 1., 1.],
         [1., 1., 1., ..., 1., 1., 1.],
         [1., 1., 1., ..., 1., 1., 1.]]]]), array([[[[1., 1., 1., ..., 1., 1., 1.],
         [1., 1., 1., ..., 1., 1., 1.],
         [1., 1., 1., ..., 1., 1., 1.],
         ...,
         [1., 1., 1., ..., 1., 1., 1.],
         [1., 1., 1., ..., 1., 1., 1.],
         [1., 1., 1., ..., 1., 1., 1.]]]])], [array([[[[1., 1., 1., ..., 1., 1., 1.],
         [1., 1., 1., ...
kwargs:    {}
Exception: TimeoutError('Timeout')

distributed.worker - WARNING -  Compute Failed
Function:  execute_task
args:      ((subgraph_callable, (subgraph_callable, (<built-in function getitem>, (<function concatenate3 at 0x2ba3975ad620>, (<function concrete at 0x2ba313031a60>, [[[[array([[[[1., 1., 1., ..., 1., 1., 1.],
         [1., 1., 1., ..., 1., 1., 1.],
         [1., 1., 1., ..., 1., 1., 1.],
         ...,
         [1., 1., 1., ..., 1., 1., 1.],
         [1., 1., 1., ..., 1., 1., 1.],
         [1., 1., 1., ..., 1., 1., 1.]]]]), array([[[[1., 1., 1., ..., 1., 1., 1.],
         [1., 1., 1., ..., 1., 1., 1.],
         [1., 1., 1., ..., 1., 1., 1.],
         ...,
         [1., 1., 1., ..., 1., 1., 1.],
         [1., 1., 1., ..., 1., 1., 1.],
         [1., 1., 1., ..., 1., 1., 1.]]]]), array([[[[1., 1., 1., ..., 0., 0., 0.],
         [1., 1., 1., ..., 0., 0., 0.],
         [1., 1., 1., ..., 0., 0., 0.],
         ...,
         [1., 1., 0., ..., 0., 0., 0.],
         [1., 1., 1., ..., 0., 0., 0.],
         [1., 1., 1., ..., 0., 0., 0.]]]])], [array([[[[1., 1., 1., ..., 1., 1., 1.],
         [1., 1., 1., ...
kwargs:    {}
Exception: TimeoutError('Timeout')

distributed.worker - WARNING -  Compute Failed
Function:  execute_task
args:      ((subgraph_callable, (subgraph_callable, (<built-in function getitem>, (<function concatenate3 at 0x2ba39754b510>, (<function concrete at 0x2ba313031a60>, [[[[array([[[[0., 0., 0., ..., 0., 0., 0.],
         [0., 0., 0., ..., 0., 0., 0.],
         [0., 0., 0., ..., 0., 0., 0.],
         ...,
         [0., 0., 0., ..., 0., 0., 0.],
         [0., 0., 0., ..., 0., 0., 0.],
         [0., 0., 0., ..., 0., 0., 0.]]]], dtype=float32), array([[[[0., 0., 0., ..., 0., 0., 0.],
         [0., 0., 0., ..., 0., 0., 0.],
         [0., 0., 0., ..., 0., 0., 0.],
         ...,
         [0., 0., 0., ..., 0., 0., 0.],
         [0., 0., 0., ..., 0., 0., 0.],
         [0., 0., 0., ..., 0., 0., 0.]]]], dtype=float32), array([[[[0., 0., 0., ..., 0., 0., 0.],
         [0., 0., 0., ..., 0., 0., 0.],
         [0., 0., 0., ..., 0., 0., 0.],
         ...,
         [0., 0., 0., ..., 0., 0., 0.],
         [0., 0., 0., ..., 0., 0., 0.],
         [0., 0., 0., ..., 0., 0., 0.]]]], dtype=float32)], [array([[[[ 0.       
kwargs:    {}
Exception: TimeoutError('Timeout')

distributed.worker - WARNING -  Compute Failed
Function:  execute_task
args:      ((subgraph_callable, (subgraph_callable, (<built-in function getitem>, (<function concatenate3 at 0x2ba3969631e0>, (<function concrete at 0x2ba313031a60>, [[[[array([[[[0.04926901, 0.0519636 , 0.05510293, ..., 0.05480079,
          0.04830868, 0.04875205],
         [0.05287233, 0.05582684, 0.05922176, ..., 0.0759612 ,
          0.06717931, 0.06693045],
         [0.05693937, 0.06022091, 0.06390154, ..., 0.1004089 ,
          0.08539097, 0.07468825],
         ...,
         [0.09264766, 0.08862296, 0.08457842, ..., 0.06311743,
          0.06455816, 0.07451491],
         [0.09240106, 0.08871574, 0.08507996, ..., 0.06939298,
          0.06884791, 0.08731735],
         [0.09214811, 0.08883138, 0.08559393, ..., 0.0627371 ,
          0.05406447, 0.05600242]]]], dtype=float32), array([[[[ 0.05954152,  0.05633717,  0.05234483, ...,  0.19526821,
           0.20617707,  0.22533473],
         [ 0.0769911 ,  0.06544954,  0.05613401, ...,  0.19192012,
           0.2046906 ,  0.21665883],
         [ 0
kwargs:    {}
Exception: TimeoutError('Timeout')

distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.6.171:57258 remote=tcp://172.30.100.1:40264>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.6.171:57246 remote=tcp://172.30.100.1:40264>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.6.171:57268 remote=tcp://172.30.100.1:40264>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.6.171:57366 remote=tcp://172.30.100.1:40264>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.6.171:57316 remote=tcp://172.30.100.1:40264>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.6.171:57320 remote=tcp://172.30.100.1:40264>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.6.171:57380 remote=tcp://172.30.100.1:40264>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.6.171:57324 remote=tcp://172.30.100.1:40264>
distributed.utils_perf - INFO - full garbage collection released 14.15 MB from 722 reference cycles (threshold: 10.00 MB)
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.6.171:57218 remote=tcp://172.30.100.1:40264>
slurmstepd: error: *** JOB 8988025 ON n2402 CANCELLED AT 2020-01-27T17:19:46 DUE TO TIME LIMIT ***
